### DISCUSSION TITLE
How to plug the sensor wires

### DISCUSSION THREAD

**Discussion by obones** [mechano/Kingroon_KP3S_Pro_Marlin#9] [Q&A] [answered]
Hello,

I have this 3D touch sensor from TriangleLabs:

![image](https://github.com/mechano/Kingroon_KP3S_Pro_Marlin_Firmware/assets/1314739/fee3ad41-6912-46b6-b76f-99856d4ba3aa)

It comes with two plugs, one with 3 wires and one with 2
According to discussion #1 it seems I have to plug the one with 2 connectors in place of the existing zMin.
But where do I plug the other one?

And isn't there  a risk of going "too low" if the existing zmin switch is disconnected? Wouldn't it be possible to plug both the black/white and original zmin in parallel?

Finally, would have any STL suggestion for a support on a stock KP3S Pro S1?

Thanks a lot for your help.

**Comment by mechano** [comment, marked as answer]
The 2 wires connector goes to Z-Max green plug. 

For the 3 wires there's a 3 pin connector named BL-Touch

![image](https://github.com/mechano/Kingroon_KP3S_Pro_Marlin_Firmware/assets/2874499/444c8a96-966b-4a80-a7b7-4a3894873ab4)
